psng/dev_environment_branch.md

```diff
@@ -0,0 +1,8 @@
+in your config dir execute:
+
+  git init
+  git remote add origin git@github.com:linuxcnc-probe-screen/probe-screen-ng.git
+  git fetch
+  git branch files_cleanup origin/files_cleanup
+  git checkout files_cleanup
```

Comment:
Better yet, you can do this:


    cd $HOME
    git clone git@github.com:linuxcnc-probe-screen/probe-screen-ng.git
    cd $HOME/linuxccnc/configs/MyConfigName
    ln -s $HOME/probe-screen-ng/macros macros-psng
    ln -s $HOME/probe-screen-ng/psng psng
    ln -s $HOME/probe-screen-ng/python python

This way, you have a clean clone of the repository without all your extra linuxcnc files showing up in `git status` etc, and can still easily make a change, then just restart linuxcnc to try it out.